src/providers/postgres/qgspostgresprovider.cpp

```diff
@@ -4759,9 +4759,9 @@ bool QgsPostgresProviderMetadata::saveStyle( const QString &uri, const QString &
   // two values are both replaced in the final .arg call of the string construction.
 
   QString sql = QString( "INSERT INTO layer_styles("
-                         "f_table_catalog,f_table_schema,f_table_name,f_geometry_column,styleName,styleQML,styleSLD,useAsDefault,description,owner%11"
+                         "f_table_catalog,f_table_schema,f_table_name,f_geometry_column,styleName,styleQML,styleSLD,useAsDefault,description%11"
                          ") VALUES ("
-                         "%1,%2,%3,%4,%5,XMLPARSE(DOCUMENT %16),XMLPARSE(DOCUMENT %17),%8,%9,%10%12"
```

Comment:
Why not just set `owner` to `current_user` instead of the uri param with current user?  If the table already exists the new default won't be set.

Reply:
Thanks @jef-n for the review.

Really helpful your suggestion! It supports old `layer_styles` tables, with the default value declared. Implemented, as suggested.